tests/lib/config/config-file.js

```diff
@@ -497,6 +519,32 @@ describe("ConfigFile", function() {
             assert.deepEqual(config, updatedConfig.eslintConfig);
         });
 
+        it("should load fresh information from a .eslintrc.js file", function() {
+            var initialConfig = {
+                    parserOptions: {},
+                    env: {},
+                    globals: {},
+                    rules: {
+                        quotes: [2, "double"]
+                    }
+                },
+                updatedConfig = {
+                    parserOptions: {},
+                    env: {},
+                    globals: {},
+                    rules: {
+                        quotes: 0
+                    }
+                },
+                tmpFilename = ".eslintrc.js",
+                tmpFilePath = writeTempJsConfigFile(initialConfig, tmpFilename),
+                config = ConfigFile.load(tmpFilePath);
+            assert.deepEqual(config, initialConfig);
+            writeTempJsConfigFile(updatedConfig, tmpFilename, path.dirname(tmpFilePath));
```

Comment:
It was using [raszi/node-tmp](https://github.com/raszi/node-tmp), the same as the existing `writeTempConfigFile` function, which is supposed to clean up the files automatically when the process exits.

I've changed it to use [bruce/node-temp](https://github.com/bruce/node-temp) which successfully cleans up the temp dir automatically.

Do you want me to amend this PR/create a new one so `writeTempConfigFile` also cleans up the temp files?